server/views/pages/bookAVisit/visitors.njk

```diff
@@ -122,26 +123,42 @@
               ],
               rows: prisonerRestrictions
             }) }}
+          {% else %}
+            {# if no restrictions, show this under the restriction header #}
+            <p data-test="restriction-description">There are no restrictions to review.</p>
           {% endif %}
+          
+          <h2 class="govuk-heading-m">Alerts</h2>
 
           {% if alerts | length %}
-          {{ govukTable({
-            classes: 'bapv-table-no-bottom-border fix-width-alert-restriction',
-            head: [
-              {
-                text: "Type of alert"
-              },
-              {
-                text: "Comments"
-              },
-              {
-                text: "End date"
-              }
-            ],
-            rows: prisonerAlerts
-          }) }}
+
+            <p data-test="alert-description">This tables shows alerts that are relevant for social visits. You can also <a class="govuk-link--no-visited-state" href="{{ prisonerDpsAlertsUrl }}" target="_blank" data-test="all-alerts-link">view all alerts (opens in new tab)</a>.</p>
+
+            {{ govukTable({
+              classes: 'bapv-table-no-bottom-border fix-width-alert-restriction',
+              head: [
+                {
+                  text: "Type of alert"
+                },
+                {
+                  text: "Comments"
+                },
+                {
+                  text: "End date"
+                }
+              ],
+              rows: prisonerAlerts
+            }) }}
+          {% else %}
+            {# if no alerts, show this under the alert header #}
+            <p data-test="alert-description">You can also <a class="govuk-link--no-visited-state" href="{{ prisonerDpsAlertsUrl }}" target="_blank" data-test="all-alerts-link">view all alerts (opens in new tab)</a>.</p>
```

Comment:
Not happy with this repeated text in 3 places (1st one just has extra text) but at the moment all I can think is to stick it in a variable at the top then use `{{ alertDescriptionText | safe }}` in these locations?